nssp_demo/prep_data.py

```diff
@@ -94,9 +177,6 @@ def process_and_save_state(
             & (pl.col("disease") == disease)
             & (pl.col("parameter") == "right_truncation")
             & (pl.col("end_date").is_null())
-            & (
-                pl.col("reference_date") <= report_date
-            )  # estimates nearest the report date
```

Comment:
Why is this removed?

Reply:
For report dates far enough in the past, there will not be a PMF with an appropriate reference date, and since we were planning not to attempt right-truncuation modeling for the most recent data, I thought simplest might be to use a fixed PMF. But we could of course always use the earliest PMF for `report_dates` before the first PMF `reference_date` (or fit PMFs for the earlier period based on the vintages! but we shouldn't try to do that IMO)

Reply:
Please document as an issue.

Reply:
#88

Reply:
We should still consider, but for now a solution with date math in 0543961